discord/reaction.py

```diff
@@ -114,6 +114,13 @@ def is_custom_emoji(self) -> bool:
         """:class:`bool`: If this is a custom emoji."""
         return not isinstance(self.emoji, str)
 
+    def is_super(self) -> bool:
+        """:class:`bool`: Whether this reaction is a super reaction.
```

Comment:
What would be more acceptable is the following:-

```suggestion
    def has_super(self) -> bool:
        """:class:`bool`: Whether this reaction has a super reaction within.
```

Meaning the end user can check if the `Reaction` has a super reaction. However one could argue we'd then also need a `has_normal` which doesn't read very well to me.